training_record/models.py

```diff
@@ -188,3 +192,20 @@ def __str__(self):
         return str(_f('{}状态于{}由{}变为{}', self.record_id, self.time,
                       self.get_pre_status_display(),
                       self.get_post_status_display()))
+
+
+class CampusEventFeedback(models.Model):
+    '''CampusEventFeedback stores text-like feedback for records.'''
+    class Meta:
+        verbose_name = _('培训活动反馈')
+        verbose_name_plural = _('培训活动反馈')
+
+    create_time = models.DateTimeField(verbose_name=_('创建时间'),
+                                       auto_now_add=True)
+    record = models.OneToOneField(Record, verbose_name=_('培训记录'),
+                                  related_name='feedback',
+                                  on_delete=models.CASCADE)
+    feedback = models.TextField(verbose_name=_('反馈内容'))
```

Comment:
Rename `feedback` to `content`, it's a little weird to retrieve content with `record.feedback.feedback`. Replace `TextField` with `CharField(max_length=120)`, since we don't allow long feedback.